api/v1/configurationpolicy_types.go

```diff
@@ -80,12 +80,12 @@ type Target struct {
 }
 
 // IsEmpty returns whether the defined Target would always return no objects.
-func (t Target) IsEmpty() bool {
+func (t *Target) IsEmpty() bool {
```

Comment:
I'd rather disable this linter. I prefer having value methods, it makes it clear that they do not mutate the state. And it's impossible to have a nil pointer dereference on a value method.